Tests/LibWeb/Text/input/navigation/history-replace-push-state-race.html

```diff
@@ -0,0 +1,10 @@
+<script src="../include.js"></script>
+<script>
+    asyncTest(done => {
+        history.replaceState({}, "hello", "history-replace-push-state-race.html");
+        history.replaceState({}, "hello", "history-replace-push-state-race.html");
+
+        println("test done!");
+        done();
```

Comment:
could you add a comment that this test is intended to check that we don't crash otherwise it might be confusing to see what we are testing here.

Reply:
sure, added a comment, thank you!